Assets/Scripts/Tags/AdultSpawnTag.cs

```diff
@@ -0,0 +1,13 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class AdultSpawnTag : MonoBehaviour
+{
+	void OnDrawGizmos()
+	{
+		Gizmos.matrix = GetComponent<Transform>().localToWorldMatrix;
+
+		Gizmos.DrawCube( new Vector3( 0.0f, 1.0f, 0.0f ), new Vector3( 1.0f, 2.0f, 1.0f ) );
+		Gizmos.DrawWireCube( new Vector3( 0.0f, 1.0f, 0.0f ), new Vector3( 1.0f, 2.0f, 1.0f ) );
```

Comment:
Yeah, these just draw a small cube that shows where the adult buddy will appear. The cube's base is where the object is located, and it's 1 x 1 wide, and 2 tall, which is roughly the size of a person.